dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details_facilitator.html.haml

```diff
@@ -0,0 +1,82 @@
+- pre_work_url = 'https://docs.google.com/document/d/1mmBcMoRKO2TVxQSEEb-SxMnhuDZUkjE06tHa-8CGeyE/edit'
```

Comment:
Is it expected that this is an `/edit` URL while the one just below is a `/pub` one?

Reply:
This is as provided in spec, and there isn't currently a "published" version of this document.  I'm asking Megan if she'd like to publish this doc too.